Code/Framework/AzCore/AzCore/Statistics/StatisticalProfilerProxySystemComponent.cpp

```diff
@@ -52,12 +52,10 @@ namespace AZ::Statistics
     ////////////////////////////////////////////////////////////////////////////////////////////////
     void StatisticalProfilerProxySystemComponent::Activate()
     {
-        m_StatisticalProfilerProxy = new StatisticalProfilerProxy;
```

Comment:
Only one instance of the statistical profiler proxy should be created. This was moved to an earlier point in application initialization in order to be able to use it for tracking budget totals.